src/main/java/org/ambraproject/wombat/service/PeerReviewService.java

```diff
@@ -0,0 +1,8 @@
+package org.ambraproject.wombat.service;
+
+import java.io.IOException;
+import java.util.Map;
+
+public interface PeerReviewService {
+  String asHtml(Map<String, ?> itemTable) throws IOException;
```

Comment:
A javadoc comment might be helpful here. I get what return type will be, however, have no idea what an item table is, well a map ... of course, would expect to look at tests for better usage, but should be usable by just inspecting this interface.